assets/scss/_footer.scss

```diff
@@ -3,17 +3,83 @@
 
 #footer {
   z-index:100;
-  padding:5vw;
-  background: $light_yellow;
+  .homepage {
+    &:before {
+      content:"";
+      background-color:#d0bca9;
+      position: absolute;
+      height: 75%;
+      width: 200vw;
+      left: -100vw;
+      z-index: -1;
+    }
+  }
+  .alt {
+    &:before {
+      content:"";
+      background-color:$light_yellow;
+      position: absolute;
+      height: 100%;
+      width: 200vw;
+      left: -100vw;
+      z-index: -1;
+    }
+  }
+
+  .homepage {
+    background: #d0bca9;
+
+    .footerinfo {
+      @media (max-width: 992px) {
+        border-right-style: none;
+      }
+      border-right: 3px solid #928477;
+    }
+    .footer-signup {
+      padding-top: 5vw;
+    }
+    .footer-signup img {
+      width: 250px;
+    }
+    .footerlogo {
+      width:250px;
+    }
+    .button {
+      border-radius: 5px;
+      background-color:white;
+      border-color: white;
+      &:hover, &.filled {
+        background: $dark_gray; color: #fff; border-color: $dark_gray;
+      }
+    }
+  }
+  .alt {
+    background: $light_yellow;
+    .footerinfo {
+      @media (max-width: 992px) {
+        border-right-style: none;
+      }
+      border-right: 3px solid $yellow;
+    }
+    .footer-signup img {
+      padding-top: 5vw;
+      width: 200px;
+    }
+    .footerlogo {
+      width: 150px;
+    }
+  }
 }
 .footer {
+  background-size:cover;
+  padding:0 5vw 10vw 5vw;
```

Comment:
Please change the padding to `padding: 0 5vw 5vw;` - 10vw at the bottom is too much.